WeaselTSF/CandidateList.cpp

```diff
@@ -423,7 +424,33 @@ void WeaselTSF::_SelectCandidateOnCurrentPage(size_t index)
 	m_client.SelectCandidateOnCurrentPage(index);
 	// fake a emptyp presskey to get data back and DoEditSession
 	m_client.ProcessKeyEvent(0);
-	_UpdateComposition(_pEditSessionContext);
```

Comment:
I mean here, send a key event instead.

There is existing logic that does `GetResponseData`, `_UpdateLanguageBar` and insert commit text. If possible, make the minimal change by reusing existing code.

When sending a key event, try something without explicit behaviour, like `VK_PROCESSKEY`.